packages/jest-preset/helpers/lastCallOfMock.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function lastCallOfMock(mockFn) {
```

Comment:
Solid helper, but there's also `toHaveBeenLastCalledWith` available as well

Reply:
I'm not sure that one helps me. I don't want to asserts the entire bag, just one property of one argument. Am I missing something?